api/nakadi-event-bus-api.yaml

```diff
@@ -1245,6 +1245,112 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /timelines:
+    post:
+      tags:
+        - timelines-api
+      description: |
+        Creates timeline and switches to it.
```

Comment:
I would say something like 'Creates a new timeline for an event type, and switches event production to the new timeline.'